package.json

```diff
@@ -181,9 +181,10 @@
     "eventsource-polyfill": "^0.9.6",
     "expect.js": "~0.2.0",
     "grunt-karma": "~0.12.1",
-    "http-proxy-middleware": "^0.17.0",
-    "inject-loader": "^3.0.0-beta4",
-    "istanbul": "^0.3.14",
+    "http-proxy-middleware": "^0.12.0",
```

Comment:
some of these dependencies have been downgraded, http-proxy-middleware and inject loader